src/test/documentation_examples/mod.rs

```diff
@@ -1755,6 +1754,72 @@ async fn index_examples() -> Result<()> {
     Ok(())
 }
 
+async fn change_streams_examples() -> Result<()> {
+    use crate::{change_stream::options::FullDocumentType, options::ChangeStreamOptions, RUNTIME};
+    use futures::stream::StreamExt;
+    use std::time::Duration;
+
+    let client = TestClient::new().await;
+    if !client.is_replica_set() && !client.is_sharded() {
+        log_uncaptured("skipping change_streams_examples due to unsupported topology");
+        return Ok(());
+    }
+    let db = client.database("change_streams_examples");
+    db.drop(None).await?;
+    let inventory = db.collection::<Document>("inventory");
+    // Populate an item so the collection exists for the change stream to watch.
+    inventory.insert_one(doc! {}, None).await?;
+
+    // Background writer thread so that the `stream.next()` calls return something.
+    let (tx, mut rx) = tokio::sync::oneshot::channel();
+    let writer_inventory = inventory.clone();
+    let handle = RUNTIME
+        .spawn(async move {
+            let mut interval = RUNTIME.interval(Duration::from_millis(100));
+            loop {
+                tokio::select! {
+                    _ = interval.tick() => {
+                        writer_inventory.insert_one(doc! {}, None).await?;
+                    }
+                    _ = &mut rx => break,
+                }
+            }
+            Result::Ok(())
+        })
+        .unwrap();
+
+    #[allow(unused_variables)]
+    {
+        // Start Changestream Example 1
+        let mut stream = inventory.watch(None, None).await?;
+        let next = stream.next().await.transpose()?;
```

Comment:
Rather than using `next()` and `transpose`, we can also import `TryStreamExt` and then just `try_next()`. I think that's what we have in our existing examples. 

On a related note, I think we might want to include the `(Try)StreamExt` import in the examples, since users will probably just copy/paste them and figuring out they need to import that trait could be difficult. That said, they'll also need to figure out they need a `futures` dependency, so maybe it doesn't really matter.

Reply:
Updated to use `TryStreamExt` and include that in the examples.  I don't think the `futures` dependency needs to be worried about; anyone more than passingly familiar with Rust will know that `use foo::bar;` means you need to pull in crate `foo`.

Reply:
Derp, done.